### ISSUE TITLE
Backend preview for nested content elements not working

### ISSUE THREAD

**Issue by nilkra** [FriendsOfTYPO3/content-blocks#128]
I am trying to use a nested content element with the following configuration:
```
name: myext/accordion
title: myext/accordion
description: "Description for Content Element myext/accordion"
group: common
prefixFields: true
prefixType: full
fields:
  - identifier: accordionItem
    type: Collection
    minitems: 1
    foreign_table: tt_content
    overrideChildTca:
      columns:
        CType:
          config:
            default: text
```
and therefore i have to change the EditorPreview. If i try to put:
```
<f:render partial="PageLayout/Grid" arguments="{data: data, identifier: 'accordionItem'}" />
```
in the EditorPreview.html of my nested content element, I get this error:
```
The Fluid template files
"/var/www/html/vendor/myvendor/myext/ContentBlocks/ContentElements/accordion/Source/PartialsPageLayout/Grid.html",
"/var/www/html/vendor/myvendor/myext/ContentBlocks/ContentElements/accordion/Source/PartialsPageLayout/Grid"
could not be loaded.
```
The Problem is the wrong partials root path. How can i fix that?

**Comment by nhovratov** [comment]
@NzudemK This feature just got merged yesterday and is only available in the main branch. You can try it out by require dev-main in composer, or you wait until the relase of 0.6.

**Comment by nhovratov** [comment]
Closing this one. This feature will be released on 26.02
